packages/preview/lucky-icml/0.2.1/icml2024.typ

```diff
@@ -0,0 +1,701 @@
+
+  // Math equation numbering and referencing.
+  set math.equation(numbering: "(1)")
+  show ref: it => {
+    let eq = math.equation
+    let el = it.element
+    if el != none and el.func() == eq {
+      let numb = numbering(
+        "1",
+        ..counter(eq).at(el.location())
+      )
+      let color = rgb(0%, 8%, 45%)  // Originally `mydarkblue`. :D
+      let content = link(el.location(), text(fill: color, numb))
+      [(#content)]
+    } else if el != none and el.func() == heading {
+      let numb = numbering(
+        el.numbering,
+        ..counter(el.func()).at(el.location()))
+      if numb.at(-1) == "." {
+        numb = numb.slice(0, -1)
+      }
+      let color = rgb(0%, 8%, 45%)  // Originally `mydarkblue`. :D
+      let content = text(fill: color, numb)
+      // If numbering starts with letter then the heading is an appendix.
+      let supplement = el.supplement
+      if numb.at(0) not in ("0", "1", "2", "3", "4", "5", "6", "7", "8", "9") {
+        supplement = [Appendix]
+      }
+      link(el.location())[#supplement~#content]
+    } else if el != none and el.func() == figure {
+      let numb = numbering(
+        el.numbering,
+        ..counter(figure.where(kind: el.kind)).at(el.location()))
+      if numb.at(-1) == "." {
+        numb = numb.slice(0, -1)
+      }
+      let color = rgb(0%, 8%, 45%)  // Originally `mydarkblue`. :D
+      let content = text(fill: color, numb)
+      link(el.location())[#el.supplement~#content]
+    } else {
+      render-ref-statement(it)
+    }
+  }
+
+  // Configure algorithm rendering.
+  counter(figure.where(kind: "algorithm")).update(0)
+  show figure.caption.where(kind: "algorithm"): it => {
+    strong[#it.supplement #it.counter.display(it.numbering)]
+    [ ]
+    it.body
+  }
+  show figure.where(kind: "algorithm"): it => {
+    place(top, float: true,
+      block(breakable: false, width: 100%, {
+        set block(spacing: 0em)
+        line(length: 100%, stroke: (thickness: 0.08em))
+        block(spacing: 0.4em, it.caption)  // NOTE: No idea why we need it.
+        line(length: 100%, stroke: (thickness: 0.05em))
+        it.body
+        line(length: 100%, stroke: (thickness: 0.08em))
+      })
+    )
+  }
+
+  // Render title.
+  text(size: font.Large, weight: "bold")[
+    #v(0.5pt)
+    #line(length: 100%)
+    #v(1pt)
+    #show par: set block(spacing: 18pt)
+    #align(center)[#title]
+    #v(1pt)
+    #line(length: 100%)
+  ]
+
+  v(0.1in - 1pt)
+
+  {
+    // Render authors.
+    set align(center)
+    make-authors().map(it => {
+      box(inset: (left: 0.5em, right: 0.5em), it)
+    }).join()
+  }
+
+  v(0.2in)
+
+  columns(2, gutter: 0.25in, {
+    set text(size: font.normal)
+    show par: set block(spacing: 11pt)
+    // Render abstract.
+    // ICML instruction tels that font size of `Abstract` must equal to 11 but
+    // it does not like so.
+    text(size: font.large)[
+      align(center)[*Abstract*]
```

Comment:
Unfortunately, I found another bug. There's a hash missing here. So it literally says `align(center)[Abstract]`.

Reply:
Thanks. Fixed.